app/helpers/filepicker_rails/application_helper.rb

```diff
@@ -258,9 +258,9 @@ def export_widget(text, url, mimetype, options, &block)
         options[:data]['fp-url'] = url
         options[:data]['fp-apikey'] = ::Rails.application.config.filepicker_rails.api_key
         options[:data]['fp-mimetype'] = mimetype
-        options[:data]['fp-option-container'] = container if container
-        options[:data]['fp-option-services'] = Array(services).join(",") if services
-        options[:data]['fp-option-defaultSaveasName'] = save_as if save_as
```

Comment:
I took a look at the [documentation](https://www.filestack.com/docs/file-ingestion/widgets/pick) and yes these `fp-option` does not exist.

I can't find the `suggestedFilename` in the documentation. But we can continue on it, changing this option.